iowrappers/redis_client.go

```diff
@@ -26,6 +27,12 @@ const (
 	NumVisitorsPrefix      = "visitor_count"
 )
 
+var RedisClientContext context.Context
+
+func init() {
+	RedisClientContext = context.Background()
```

Comment:
why use a `Background` and not any other type of contexts ?

Reply:
because at the moment we are not doing anything except satisfying the function signature requirement